src/book/03-guides/03-hardware/03-batteries.mdx

```diff
@@ -0,0 +1,101 @@
+   aware that the battery monitor will beep several times when this happens, this is normal. ![Battery monitor attached to a battery](./images/batteries/battery_monitor.jpg)
+
+2. Once the battery monitor is attached to the battery, the other cable on the battery can now be plugged into the robot. Depending on the battery this may require an
+   adaptor lead, shown below.
+
+3. Once both cables on the battery are hooked up, install the battery into the robot and install the back panel cover. Be sure to carefully load the battery into the robot to avoid damage, this may
+   require some technique so don't be afraid to ask a team member for the best way to go about this.
+
+4. The robot can now be safely turned on.
+
+![Battery in robot with a battery monitor](./images/batteries/battery_in_robot.jpg)
+
+## Charging
+
+<Alert type="warning">If at any point during these steps the charging interface differs from what is shown in these photos, be sure to ask a team member for assistance to ensure your safety and to look after the battery</Alert>
+
+We have two types of chargers in the lab to charge LiPo batteries. Both achieve the same purpose and have a similar user interface. The Powertech Plus charger will be
+used for the purposes of the images below.
+
+### Charging a battery
+
+1. Plug the battery cable with the white plug into the corresponding socket on the side of the charger. The plug should only fit in one orientation so check before
+   plugging it in. If you find yourself forcing the plug into the socket you are doing something wrong.
+
+2. Plug the other battery cable into the main output cable of the charger, this may require an adaptor, as seen in the section on using in the robot
+
+3. Plug the Powertech Plus into a power socket, and use the arrow buttons to select the "Charging" setting for the type of battery you are charging. For the NUgus robot
+   battery this is a 14.8V 4 Cell, as seen in the image below. For a Darwin robot battery, this is a 11.1V 3 Cell.
+
+4. Hold down the start button until the device beeps, and observe the content on the screen. If the charging device indicates any errors, stop charging immediately,
+   unplug the battery, and return it to a fire retardant case. If there are no errors, press start to commence charging
+
+5. Close and lock the white BAT-SAFE box with the battery inside (Keep the charger outside if possible to hear when it is done)
+
+6. Stay in the lab with the battery and disconnect immediately when charge has completed. Never leave a battery charging when there is no one in the lab
+
+![Charging setting on Powertech Plus LiPo Charger](./images/batteries/charge.jpg)
+
+## Storing and Traveling with batteries
+
+<Alert>
+
+Care should be taken to verify the LiPo battery limits of the airline you are travelling with. They will usually specify a maximum `Ah` value for all batteries and
```

Comment:
They calculate in `Wh` not `Ah`, which is calculated using `Voltage * Ah`

Reply:
https://www.qantas.com/au/en/travel-info/baggage/dangerous-goods/spare-batteries-and-powerbanks.html#:~:text=Note%3A%20Watt%20hours%20(Wh)%20are%20determined%20by%20multiplying%20the%20voltage%20(V)%20by%20the%20amp%20hours%20(Ah).%20ie.%2012V%20x%205Ah%20%3D%2060Wh

Reply:
I distinctly remember being very confused when they asked for Watt Hours since it's such a weird unit lol